Makefile

```diff
@@ -122,8 +122,8 @@ acceptance:
 check:
 	@echo "checking for tabs in shell scripts"
 	@! git grep -F '	' -- '*.sh'
-	@echo "checking for \"path\" imports"
-	@! git grep -F '"path"' -- '*.go'
+	@echo "checking for forbidden imports"
+	@! go list -f '{{ $$ip := .ImportPath }}{{ range .Imports}}{{ $$ip }}: {{ println . }}{{end}}' $(PKG) | grep -E ' (path|log)$$' | grep -Ev 'util/(log|stop): log'
```

Comment:
You're right; I can't think of a case where it's necessary. Btw, I think we want `$$` on the negative match, in case we ever disallow an import called "logger", for instance